assets/queries/azureResourceManager/not_recommended_log_profile_retention_policy/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy[result] {
+	doc := input.document[i]
+	[path, value] = walk(doc)
+
+	value.type == "microsoft.insights/logprofiles"
+	value.properties.retentionPolicy.enabled == false
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": "resources.type={{microsoft.insights/logprofiles}}.properties.retentionPolicy.enabled",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "resource with type 'microsoft.insights/logprofiles' has 'enabled' property set to true",
+		"keyActualValue": "resource with type 'microsoft.insights/logprofiles' doesn't have 'enabled' set to true",
+	}
+}
+
+CxPolicy[result] {
+	doc := input.document[i]
+	[path, value] = walk(doc)
+
+	value.type == "microsoft.insights/logprofiles"
+
+	value.properties.retentionPolicy.days <= 365
```

Comment:
should we check if it's integer or string? and if it's string convert to integer

Reply:
I do not think it is necessary since `value.properties.retentionPolicy.days` is a integer